tests/regression/22-partitioned_arrays/20-more-fixpoint.c

```diff
@@ -0,0 +1,37 @@
+// PARAM: --set ana.base.arrays.domain partitioned 
+#include <stdio.h>
+#include <stdlib.h>
+#include <pthread.h>
+#include <assert.h>
+
+pthread_mutex_t m;
+
+int arr[2];
+void g() {
+  int x = arr;
```

Comment:
gcc produces a warning about this line:
```
warning: initialization of ‘int’ from ‘int *’ makes integer from pointer without a cast [-Wint-conversion]
```

Is this intended?

Reply:
No, that was by mistake, the assignment to `x` is not important here.